tools/DfmHttpService/DfmHttpService.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmHttpService
+{
+    using System;
+    using System.IO;
+    using System.Net;
+    using System.Text;
+
+    using Microsoft.DocAsCode.Build.Engine;
+    using Microsoft.DocAsCode.Plugins;
+
+    using Newtonsoft.Json;
+
+    public class DfmHttpService
+    {
+        private bool KeepGoing { get; set; } = true;
```

Comment:
Any reason to use private property here? Seems no necessary to expose and calculate here. Refer to [http://stackoverflow.com/questions/3310186/are-there-any-reasons-to-use-private-properties-in-c](http://stackoverflow.com/questions/3310186/are-there-any-reasons-to-use-private-properties-in-c)

---
In reply to: [94037206](https://github.com/dotnet/docfx/pull/1075#discussion_r94037206) [](ancestors = 94037206)